src/mongocxx/database.cpp

```diff
@@ -89,12 +94,18 @@ bsoncxx::document::value database::run_command(bsoncxx::document::view_or_value
     return reply_bson.steal();
 }
 
+bsoncxx::document::value database::coll_mod(stdx::string_view name,
```

Comment:
Is this function name mandated by the crud spec? Some other spec? If we have the freedom to name it differently, it seems like we should. `modify_collection`?